### ISSUE TITLE
Screenshots on the homepage/github page?

### ISSUE THREAD

**Issue by rubyFeedback** [Phaengris/PasswordStore#1]
Hey there Phaengris,

Andy mentioned glimte recently and the password store:

See here:

https://andymaleh.blogspot.com/2023/01/glimmer-ecosystem-glimte-passwordstore.html

I checked the website he mentioned, and then the github page here. I did not see a 
screenshot though. Andy uses quite a lot of screenshots for the glimmer-ecosystem,
so if you have some time, could you consider adding a screenshot too? Does not have
to be polished, just something for curious folks to quickly see where the project was
at (at the least at some point in time). Thank you for reading this.

**Comment by Phaengris** [comment]
Yes, that's probably a good idea. This project is very alpha, that's why no description / screenshots yet. As soon as I have time, I'll do it. Thanks.
